OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -81,27 +128,51 @@ public class MissileInfo : IProjectileInfo
 		public IEffect Create(ProjectileArgs args) { return new Missile(this, args); }
 	}
 
+	// TODO: double check square roots!!!
 	public class Missile : IEffect, ISync
 	{
+		enum States
+		{
+			Freefall,
+			Homing,
+			Hitting
+		}
+
 		readonly MissileInfo info;
 		readonly ProjectileArgs args;
 		readonly Animation anim;
 
+		// NOTE: Might be desirable to unhardcode the number -10
+		readonly WVec gravity = new WVec(0, 0, -10);
```

Comment:
If you do unhardcode this in a follow-up (I like the idea), personally I'd suggest leaving the minus here and only unhardcode the number, I think that would be more intuitive for modders (higher value = stronger gravity).